### PR TITLE
Fix Autofill errors 

### PR DESCRIPTION
**Pull request by CoderHam**
Follow up to earlier autofill fix - model config test failure resolved

### CODE REVIEW

File: qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected_2 [hunk 1]
```diff
@@ -0,0 +1 @@
+ensemble tensor 'temp_tensor_5' is unused in ensemble 'tensor_to_input_overmapped'
```
Comment (GuanLuo) [inline qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected_2]: Why touching ensemble test? Was this removed accidentally in earlier changes?

Comment (CoderHam) [reply qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected_2]: Very likely. This was not failing due to my changes. The error message expected itself was incorrect - i.e. the one in qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected

Comment (GuanLuo) [reply qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected_2]: There is no reason to keep expected as the error message is changed, you should just modify expected and remove the empty expected_2.

Comment (CoderHam) [reply qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected_2]: Sure

File: src/backends/tensorflow/autofill.cc [hunk 2]
```diff
@@ -330,6 +330,11 @@ AutoFillGraphDef::Create(
   // attempt to determine the platform. For now we allow multiple versions
   // and only inspect the first verison directory to ensure it is valid.
   // We can add more aggressive checks later.
+  if (version_dirs.size() == 0) {
+      return Status(
+          RequestStatusCode::INTERNAL, "unable to autofill for '" + model_name +
```
Comment (dzier) [inline src/backends/tensorflow/autofill.cc]: I think you need to run the formatter on this file

Comment (CoderHam) [reply src/backends/tensorflow/autofill.cc]: Done

**Pull request merged**